waspc/data/Generator/templates/react-app/src/actions/index.js

```diff
@@ -1,7 +1,7 @@
 import {
   useMutation,
   useQueryClient,
-} from 'react-query'
+} from '@tanstack/react-query'
```

Comment:
Reference: https://tanstack.com/query/v4/docs/guides/migrating-to-react-query-4#react-query-is-now-tanstackreact-query